src/OpenLoco/Core/Variant.hpp

```diff
@@ -0,0 +1,21 @@
+// This file enables access to std::visit
+
+#pragma once
+
+#if !defined(__APPLE__)
```

Comment:
Can we add a TODO here about using Cmake to detect std::variant support? Could also be a hacktoberfest issue.

Reply:
Hmm, sure. It's a pity `visit` is defined in the `variant` header. macOS 10.13 does ship with `variant`, just not with `visit`. Otherwise, we could've just used that as a hint…